h2d/Layers.hx

```diff
@@ -35,6 +51,10 @@ class Layers extends Object {
 			layersIndexes[i]++;
 	}
 
+	/**
+	 * Removes a child `h2d.Object`.
+	 * @param s `h2d.Object` to be removed.
+	 */

```

Comment:
Given it's an override this does not require documentation (part of h2d.Object)

Reply:
Removed docs that I added on overriden methods.